tests/components/ozw/test_climate.py

```diff
@@ -0,0 +1,131 @@
+"""Test Z-Wave Multi-setpoint Climate entities."""
+from homeassistant.components.climate import ATTR_TEMPERATURE
+from homeassistant.components.climate.const import (
+    ATTR_CURRENT_TEMPERATURE,
+    ATTR_FAN_MODE,
+    ATTR_FAN_MODES,
+    ATTR_PRESET_MODES,
+    ATTR_TARGET_TEMP_HIGH,
+    ATTR_TARGET_TEMP_LOW,
+    HVAC_MODE_AUTO,
+    HVAC_MODE_HEAT,
+)
+
+from .common import setup_ozw
+
+
+async def test_climate(hass, climate_data, sent_messages, climate_msg):
+    """Test setting up config entry."""
+    receive_message = await setup_ozw(hass, fixture=climate_data)
+
+    # Test multi-setpoint thermostat (node 7 in dump)
+    # mode is heat, this should be single setpoint
+    state = hass.states.get("climate.ct32_thermostat_mode")
+    assert state is not None
+    assert state.state == HVAC_MODE_HEAT
+    assert state.attributes[ATTR_CURRENT_TEMPERATURE] == 23.1
+    assert state.attributes[ATTR_TEMPERATURE] == 21.1
+    assert state.attributes.get(ATTR_TARGET_TEMP_LOW) is None
+    assert state.attributes.get(ATTR_TARGET_TEMP_HIGH) is None
+    assert state.attributes[ATTR_FAN_MODE] == "Auto Low"
+    assert state.attributes[ATTR_FAN_MODES] == ["Auto Low", "On Low"]
+
+    # Test set target temperature
+    await hass.services.async_call(
+        "climate",
+        "set_temperature",
+        {"entity_id": "climate.ct32_thermostat_mode", "temperature": 26.1},
+        blocking=True,
+    )
+    assert len(sent_messages) == 1
+    msg = sent_messages[0]
```

Comment:
If we get the last message in the list, the diff will be smaller if we extend the test in the future.

Reply:
```suggestion
    msg = sent_messages[-1]
```